CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,58 @@
+- [Italian/Italiano](code-of-conduct-languages/it.md)
+- [Japanese/日本語](code-of-conduct-languages/jp.md)
+- [Korean/한국어](code-of-conduct-languages/ko.md)
+- [Ukrainian/Українська](code-of-conduct-languages/uk.md)
+- [Russian/Русский](code-of-conduct-languages/ru.md)
+- [Portuguese/Português](code-of-conduct-languages/pt.md)
+- [Arabic/العربية](code-of-conduct-languages/ar.md)
+- [Polish/Polski](code-of-conduct-languages/pl.md)
+- [Indonesian/Bahasa Indonesia](code-of-conduct-languages/id.md)
+
+### Contributor Code of Conduct
+
+As contributors and maintainers of this project, and in the interest of fostering
+an open and welcoming community, we pledge to respect all people who contribute
+through reporting issues, posting feature requests, updating documentation,
+submitting pull requests or patches, and other activities.
+
+We are committed to making participation in this project a harassment-free experience for
+everyone, regardless of level of experience, gender, gender identity and expression,
+sexual orientation, disability, personal appearance, body size, race, ethnicity, age,
+religion, or nationality.
+
+Examples of unacceptable behavior by participants include:
+
+* The use of sexualized language or imagery
+* Personal attacks
+* Trolling or insulting/derogatory comments
+* Public or private harassment
+* Publishing others' private information, such as physical or electronic addresses,
+ without explicit permission
+* Other unethical or unprofessional conduct.
+
+Project maintainers have the right and responsibility to remove, edit, or reject
+comments, commits, code, wiki edits, issues, and other contributions that are not
+aligned to this Code of Conduct. By adopting this Code of Conduct, project maintainers
+commit themselves to fairly and consistently applying these principles to every aspect
+of managing this project. Project maintainers who do not follow or enforce the Code of
+Conduct may be permanently removed from the project team.
+
+This code of conduct applies both within project spaces and in public spaces
+when an individual is representing the project or its community.
+
+Instances of abusive, harassing, or otherwise unacceptable behavior in Kubernetes may be reported by contacting the [Kubernetes Code of Conduct Committee](https://git.k8s.io/community/committee-code-of-conduct) via <conduct@kubernetes.io>. For other projects, please contact a CNCF project maintainer or our mediator, Mishi Choudhary <mishi@linux.com>.
+
+This Code of Conduct is adapted from the Contributor Covenant
+(http://contributor-covenant.org), version 1.2.0, available at
+http://contributor-covenant.org/version/1/2/0/
+
+### CNCF Events Code of Conduct
+
+CNCF events are governed by the Linux Foundation [Code of Conduct](https://events.linuxfoundation.org/code-of-conduct/) available on the event page. This is designed to be compatible with the above policy and also includes more details on responding to incidents.
```

Comment:
This file should point to the CNCF CoC, not copy it.

See:
- https://github.com/fluxcd/flux/blob/master/CODE_OF_CONDUCT.md
- https://github.com/fluxcd/toolkit/blob/main/CODE_OF_CONDUCT.md
- https://github.com/fluxcd/helm-controller/blob/main/CODE_OF_CONDUCT.md

And other CNCF projects:
- https://github.com/kubernetes/community/blob/master/code-of-conduct.md
- https://github.com/helm/community/blob/master/code-of-conduct.md


Essentially, this:
```suggestion
# Community Code of Conduct

Flux follows the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md).
```